app/models/katello/glue/candlepin/candlepin_object.rb

```diff
@@ -0,0 +1,52 @@
+module Katello
+  module Glue::Candlepin::CandlepinObject
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def get_for_organization(organization)
+        # returns objects from AR database rather than candlepin data
+        pool_ids = self.get_for_owner(organization.label).collect { |x| x['id'] }
+        self.where(:cp_id => pool_ids)
+      end
+
+      def get_candlepin_ids(organization)
+        self.get_for_owner(organization).map { |subscription| subscription["id"] }
+      end
+
+      def import_candlepin_ids(organization)
+        candlepin_ids = self.get_candlepin_ids(organization)
+        candlepin_ids.each do |cp_id|
+          self.find_or_create_by_cp_id(cp_id) unless cp_id.nil?
```

Comment:
possibly another rails3 specific method; however I do remember seeing stuff related to candlepin where theses types of methods were overridden.